lib/countly_flutter.dart

```diff
@@ -1951,6 +1962,9 @@ class Countly {
       if (config.loggingEnabled != null) {
         countlyConfig['loggingEnabled'] = config.loggingEnabled;
       }
+      if (config.locationDisabled) {
```

Comment:
can you re verify if this should not be: (config.locationDisabled != null)

Reply:
Yeah, locationDisabled is not nullable, so it can't be null. Also, the default value is false.